src/components/layout/FixedLeft/FixedLeft.jsx

```diff
@@ -74,6 +75,8 @@ class FixedLeft extends React.Component<Props, State> {
   fixedViewportOffsetOrigin: number;
 
   render() {
+    const children = React.Children.toArray(this.props.children);
```

Comment:
what's happening here?

Reply:
It's correct, according to the docs.

Reply:
Same as above comment: https://github.com/wework/plasma/pull/260#discussion_r316150721